docs/pages/using-nhs-notify/api.md

```diff
@@ -0,0 +1,17 @@
+---
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: page
+title: API
+parent: Using NHS Notify
+nav_order: 1
+permalink: /using-nhs-notify/api
+---
+
+NHS England organisations and services can integrate the NHS Notify API with their web applications or back office systems.
+
+<a href="https://digital.nhs.uk/developer/api-catalogue/nhs-notify" target="_blank">Read our API documentation (opens in a new tab)</a>.
```

Comment:
Use a markdown url for this, not href

Reply:
Replaced with Markdown equivalent - noob question, why is using in-line HTML bad practice in Markdown?